build-system/tasks/dep-check.js

```diff
@@ -203,7 +203,9 @@ function getGraph(entryModule) {
  */
 function getEntryModule(extensionFolder) {
   // TODO (@zhouyx, #9642): Remove the special check and handle more than just 0.1
-  if (extensionFolder == 'extensions/amp-sticky-ad') {
+  if (extensionFolder == 'extensions/amp-sticky-ad' ||
```

Comment:
You can't do this for `amp-sidebar`. `amp-sticky-ad` is OK because 0.1 is fully deprecated.